RazorGenerator.Testing/WebViewPageExtensions.cs

```diff
@@ -24,28 +24,35 @@ public static class WebViewPageExtensions
 
         private static readonly Dictionary<string, ViewPlaceholder> _views = new Dictionary<string, ViewPlaceholder>();
 
-        public static IViewEngine ViewEngine
+        private static StringWriter _writer;
+
+        public static DummyViewEngine ViewEngine
         {
             get { return _viewEngine; }
         }
 
+        public static StringWriter Writer
```

Comment:
Wouldn't this cause issues when you run tests in parallel (I know xunit does this by default)?